ui/apps/platform/src/routePaths.js

```diff
@@ -163,7 +163,8 @@ const vulnerabilitiesPathToLabelMap = {
 
 export const basePathToLabelMap = {
     [dashboardPath]: 'Dashboard',
-    [networkBasePath]: 'Network Graph (2.0)',
+    [networkBasePath]: 'Network Graph',
+    [listeningEndpointsBasePath]: 'Listening Endpoints',
```

Comment:
Although better safe than sorry for now, we want to simplify away from this step.